src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -48,13 +48,19 @@ static ScannedDocument createScannedDocumentWithCcdData(Map<String, Object> obje
     }
 
     private static Document mapScannedDocument(ScannedDocument scannedDocument) {
-        return new Document(
-            scannedDocument.fileName,
-            scannedDocument.controlNumber,
-            scannedDocument.type,
-            scannedDocument.subtype,
-            scannedDocument.scannedDate.atZone(ZoneId.systemDefault()).toInstant(),
-            scannedDocument.url.documentUrl
-        );
+        if (scannedDocument == null) {
```

Comment:
This would mean ccd is sending us a list of documets containing null items. Could we validate the list and returns etror message to ccd instead of skipping?

Reply:
Whether existing docs have this date should not matter to use, we're just adding more documents.